packages/core/src/features/st-import.ts

```diff
@@ -90,7 +90,8 @@ export const diagnostics = {
     UNKNOWN_IMPORTED_FILE: createDiagnosticReporter(
         '05016',
         'error',
-        (path: string) => `cannot resolve imported file: "${path}"`
+        (path: string, error?: unknown) =>
```

Comment:
why is the error optional? it's only called in one place that has an error to pass

Reply:
there are tests that does not concern that and not passing error here...they check with `include`

Reply:
ok